src/components/calltree/CallTree.js

```diff
@@ -164,7 +164,11 @@ class CallTreeComponent extends PureComponent<Props> {
       newExpandedCallNodeIndexes.push(currentCallNodeIndex);
     }
     this._onExpandedCallNodesChange(newExpandedCallNodeIndexes);
-    this._onSelectedCallNodeChange(currentCallNodeIndex);
+
+    const category = tree.getDisplayData(currentCallNodeIndex).categoryName;
+    if (category !== 'Idle') {
+      this._onSelectedCallNodeChange(currentCallNodeIndex);
```

Comment:
Can you add a comment here explaining why we are doing this? Especially with the fact that the thread activity graph is dimmed out and confusing.

Reply:
done, thanks!